aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -482,6 +482,14 @@ In previous versions of ASP.NET Core, calling <xref:Microsoft.AspNetCore.Builder
 
 This scenario is fixed in ASP.NET Core 3.0. The host enables the [Forwarded Headers Middleware](xref:host-and-deploy/proxy-load-balancer#forwarded-headers-middleware-options) when the `ASPNETCORE_FORWARDEDHEADERS_ENABLED` environment variable is set to `true`. `ASPNETCORE_FORWARDEDHEADERS_ENABLED` is set to `true` in our container images.
 
+## Performance improvements across the stack
+
+As part of ASP.NET Core 3.0, we've made improvements in reducing memory allocations and improving throughput through various parts of the stack including, but not limited to:
+- Reduction in memory usage when using the built-in Dependency Injection container for scoped services
+- Reduction in allocations across the stack including middleware scenarios and routing.
+- Reduction in memory usage for WebSocket connections.
+- Memory reduction and throughput improvements for HTTPS connections.
```

Comment:
- New optimized and fully asynchronous JSON serializer.
- Huge memory and throughout improved form parsing.